language-extensions/dotnet-core-CSharp/sample/regex/README.md

```diff
@@ -22,6 +22,11 @@ RECONFIGURE WITH OVERRIDE;`.
 
 + The **dotnet-core-CSharp-lang-extension.zip** file from the [.NET Core C# Language Extension](../../../../README.md).
```

Comment:
Could you please remove this .zip from the Prerequisites section now that we mention it explicitly below in Download required files?

Reply:
Fixed